utils/update-db.js

```diff
@@ -276,8 +276,10 @@ class KanjiDictParser extends Writable {
     //   𡗗 X253E U215D7 B37 S5 Ypeng3
     //   𢦏 .=.=== U2298F B62 S6 Yzai1 {to cut} {wound} {hurt}
     //   𢈘 .=.=== U22218 B53 S9 N1510 DP3845 ロク しか か
+    //   𪃹 .=.=== U2A0F9 B196 S20 じない {thrush} {{kokuji}}
     //
     // (Yes, including the trailing spaces too.)
+    // (The kokuji are usually rendered as {(kokuji)} though Jim says he added them as {{kokuji}} to the readings which doesn't seem true anymore)
```

Comment:
Rephrase comment to be more concise

Reply:
Not sure if it's more concise but I moved the comment closer to the example and made it more explicit.
Thanks!